devDocs/conf.py

```diff
@@ -12,6 +12,17 @@
 sys.path.insert(0, os.path.abspath('../source'))
 import sourceEnv  # noqa: F401, E402
 
+
+# Apply several monkey patches to comtypes.
+# Add our `comInterfaces` to the `comtypes.gen` seach path to replicate the behavior at runtime
+# without this patch many modules aren't importable, since they depend on `comInterfaces` being present.
+# Also disable module being never than typelib error, seen when
+# virtual environment has been created under different version of Windows than the one
+# used for developer documentation build. This also matches the behavior at runtime.
```

Comment:
I had difficulty understanding what you mean by "Also disable module being never than typelib error"

Reply:
By default when trying to use the generated com interface `comtypes complains` when it was created under the version of Windows different than the one now used. Since this complaint is harmless we just disable it with an appropriate monkey patch. Given this explanation could you please suggest appropriate wording which would be sufficient for someone not familiar with this problem to understand what I mend?

Reply:
Please confirm this reworded description is accurate.

```suggestion
# When a virtual environment has been created under a different version of Windows than the one
# used for developer documentation build, "ImportError: Typelib different than module" is raised
# by comTypes.
# This patch causes the error to be ignored, which matches the behavior at runtime.
```